crates/rspack_binding_api/src/fs_node/node.rs

```diff
@@ -33,6 +33,8 @@ pub struct ThreadsafeNodeFS {
   pub stat: ThreadsafeFunction<String, Promise<Either<NodeFsStats, ()>>>,
   #[napi(ts_type = "(name: string) => Promise<NodeFsStats | void>")]
   pub lstat: ThreadsafeFunction<String, Promise<Either<NodeFsStats, ()>>>,
+  #[napi(ts_type = "(name: string, mode: number) => Promise<NodeFsStats | void>")]
```

Comment:
is the type annotation right ?
```suggestion
  #[napi(ts_type = "(name: string, mode: number) => Promise<void>")]
```